src/components/input/input-default-styles.jsx

```diff
@@ -168,6 +168,10 @@ export default theme => {
         '&:focus': {
           outline: 'none',
         },
+        '&--textarea': {
+          maxHeight: 'none',
+          resize: 'vertical',
```

Comment:
`resize` is not supported in IE - https://developer.mozilla.org/en-US/docs/Web/CSS/resize